Header header
int8 hands_count
int8 fingers_count
geometry_msgs/Vector3 arm_direction
geometry_msgs/Point elbow
geometry_msgs/Point wrist
geometry_msgs/Vector3 hand_direction
geometry_msgs/Vector3 palm_normal
geometry_msgs/Point palmpos
geometry_msgs/Point thumb_metacarpal
geometry_msgs/Point thumb_proximal
geometry_msgs/Point thumb_intermediate
geometry_msgs/Point thumb_distal
geometry_msgs/Point thumb_tip
geometry_msgs/Point index_metacarpal
geometry_msgs/Point index_proximal
geometry_msgs/Point index_intermediate
geometry_msgs/Point index_distal
geometry_msgs/Point index_tip
geometry_msgs/Point middle_metacarpal
geometry_msgs/Point middle_proximal
geometry_msgs/Point middle_intermediate
geometry_msgs/Point middle_distal
geometry_msgs/Point middle_tip
geometry_msgs/Point ring_metacarpal
geometry_msgs/Point ring_proximal
geometry_msgs/Point ring_intermediate
geometry_msgs/Point ring_distal
geometry_msgs/Point ring_tip
geometry_msgs/Point pinky_metacarpal
geometry_msgs/Point pinky_proximal
geometry_msgs/Point pinky_intermediate
geometry_msgs/Point pinky_distal
geometry_msgs/Point pinky_tip

